nifstd/nifstd_tools/parcellation/paxinos.py

```diff
@@ -105,30 +105,34 @@ class PaxRatAt(Atlas):
                        version='7th Edition',)
 
     class PaxSpinalAt(Atlas):
-        iri = ilx['paxinos/uris/spina']  # ilxtr.paxinosMouseAtlas
+        iri = ilx['paxinos/uris/spinal']  # ilxtr.paxinosMouseAtlas
         class_label = 'Paxinos Spinal Atlas'
 
     PaxSpinalAtlas = Atlas(iri=PaxSpinalAt.iri,
-                           species=NCBITaxon['???'],
+                           species=NCBITaxon['10116'], # todo change me
```

Comment:
Which species are we focusing for this one or should I make this a tuple of species?

Reply:
Added tuple for NCBITaxon IDs reflecting: rat, mouse, marmoset, rhesus, or human. It didn't seem to help or hurt the output file.